packages/app-bridge/src/AppBridgeBlock.ts

```diff
@@ -29,6 +29,10 @@ export interface AppBridgeBlock extends AppBridgeBase {
 
     addAssetIdsToBlockAssetKey(key: string, assetIds: number[]): Promise<void>;
 
+    /**
+     * @deprecated Use dispatch(`openAssetViewer`) command instead
+     * This will be removed in version 4.0.0 of @frontify/app-bridge
+     */
```

Comment:
Keep it while making the implementation run `dispatch("openAssetViewer")` so no breaking changes